### PR TITLE
perf(preprocessing): better performance

### PR DESCRIPTION
**Pull request by 34j**

### CODE REVIEW

**Comment by 34j** [comment]
@allcontributors GarrettConway code

**Comment by 34j** [comment]
@allcontributors add GarrettConway code

**Comment by 34j** [comment]
@allcontributors add 34j example, infra, maintenance, review, test, tutorial, promotion, bug

File: src/so_vits_svc_fork/preprocess_hubert_f0.py [hunk 1]
```diff
@@ -17,41 +17,70 @@
...
+        LOG.info(f"Skip {filepath} because {soft_path} exists.")
+    f0_path = filepath.parent / (filepath.name + ".f0.npy")
+    if not f0_path.exists() or force_rebuild:
+        f0 = utils.compute_f0_dio(
+            wav, sampling_rate=sampling_rate, hop_length=hop_length
+        )
+        np.save(f0_path, f0)
+    else:
+        LOG.info(f"Skip {filepath} because {f0_path} exists.")
+    torch.cuda.empty_cache()
```
Comment (GarrettConway) [inline src/so_vits_svc_fork/preprocess_hubert_f0.py:47]: Nice. Thank you for finding that.

**Pull request merged**